### PR TITLE
feat: implemented basic side option for USDT coin

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
With this PR we are about to cover implementation of USDT

## Testing Steps (optional)
login to wallet
make sure that usdt option is visible in left navigation

### CODE REVIEW

**Comment by milan-bc** [comment]
@plondon I'll close this one, so we can move from common branch to new PR.

File: packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts [hunk 1]
```diff
@@ -2010,6 +2010,7 @@ type MessagesType = {
   'scenes.transaction.headertext.explainer.btc1': 'Bitcoin (BTC) is the original crypto and the internet’s digital currency.'
   'scenes.transaction.headertext.explainer.eth': 'Ethereum (ETH) is a currency and computing platform. Built for developers and apps.'
   'scenes.transaction.headertext.explainer.usdd': 'The USD Digital coin (USD-D) is backed by the US Dollar, making it a Stablecoin.'
+  'scenes.transaction.headertext.explainer.usdt': 'The USD Theter coin (USDT) is backed by the US Dollar, making it a Stablecoin.'
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts:2013]: Thether => Tether

File: packages/blockchain-wallet-v4-frontend/src/layouts/Wallet/MenuLeft/Navigation/template.tsx [hunk 2]
```diff
@@ -95,8 +95,7 @@ const Navigation = (props: OwnProps & Props) => {
         mapObjIndexed(
           (coin: SupportedCoinType, i) =>
             coin &&
-            coin.txListAppRoute &&
-            coin.invited && (
+            coin.txListAppRoute && (
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/layouts/Wallet/MenuLeft/Navigation/template.tsx:98]: Add back coin.invited

File: packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/template.headerexplainer.tsx [hunk 3]
```diff
@@ -105,6 +105,27 @@ export const getHeaderExplainer = coinModel => {
         </ExplainerText>
       )
     }
+    case 'USDT': {
+      return (
+        <ExplainerText>
+          <FormattedMessage
+            id='scenes.transaction.headertext.explainer.usdt'
+            defaultMessage='The USD Theter coin (USDT) is backed by the US Dollar, making it a Stablecoin.'
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/template.headerexplainer.tsx:113]: Thether => Tether

**Comment by plondon** [comment]
LGTM! A few small tweaks requested

**Pull request merged**